examples/applications/restapi/app.py

```diff
@@ -39,10 +44,21 @@ def setup_app(self) -> None:
         # app_state.daemon is initialised after app. Setup things dependent on daemon here.
         self.logger.debug("setting up daemon-app")
         self.restapi = ExtensionEndpoints()
+
+        # state management of the websocket must be via the aiohttp.web.Application object
+        # access from TxStateWebSocket View via self.request.app['ws_clients'] and
+        self.aiohttp_web_app = self.app_state.daemon.rest_server.app
+        self.aiohttp_web_app['ws_clients'] = {}  # uuid: WSClient
+        self.aiohttp_web_app['tx_registrations_map'] = {}  # tx_hash: {set of websocket uuids}
+        self.aiohttp_web_app['restapi'] = self.restapi
         self.app_state.daemon.rest_server.register_routes(self.restapi)
 
     def _teardown_app(self) -> None:
-        pass
+        try:
+            for client in self.aiohttp_web_app['ws_clients'].values():
+                asyncio.run_coroutine_threadsafe(client.websocket.close(), app_state.async_.loop)
+        except Exception:
```

Comment:
If you catch the base class Exception please put a comment why not specific exceptions.